macantine/tasks.py

```diff
@@ -92,3 +92,49 @@ def no_canteen_second_reminder():
         except Exception as e:
             logger.error(f"Unable to send second no-cantine reminder email to {user.username}")
             logger.exception(e)
+
+
+@app.task()
+def no_diagnostic_first_reminder():
+    if not settings.TEMPLATE_ID_NO_DIAGNOSTIC_FIRST:
+        logger.error("Environment variable TEMPLATE_ID_NO_DIAGNOSTIC_FIRST not set")
+        return
+    today = timezone.now()
+    threshold = today - datetime.timedelta(weeks=1)
+    canteens = Canteen.objects.filter(
+        diagnostic__isnull=True,
+        creation_date__lte=threshold,
+        email_no_diagnostic_first_reminder__isnull=True,
+    ).all()
+    if not canteens:
+        logger.info("no_diagnostic_first_reminder: No canteens to notify.")
+        return
+
+    logger.info(f"no_diagnostic_first_reminder: {len(canteens)} canteens to notify.")
+
+    for canteen in canteens:
+        for manager in canteen.managers.all():
+
+            try:
+                parameters = {"PRENOM": manager.first_name, "NOM_CANTINE": canteen.name}
+                _send_sib_template(
+                    settings.TEMPLATE_ID_NO_DIAGNOSTIC_FIRST,
+                    parameters,
+                    manager.email,
+                    f"{manager.first_name} {manager.last_name}",
+                )
+                logger.info(
+                    f"First no-diagnostic email sent to {manager.get_full_name()} ({manager.email}) for canteen '{canteen.name}'."
+                )
+                canteen.email_no_diagnostic_first_reminder = today
```

Comment:
Ça peut sembler bizarre de positionner ces lignes à l'intérieur du `for`. En gros, je considère que si on a réussi a contacter un gestionnaire, on considère que la cantine a été notifiée. Si je sors ces lignes du `for`, et qu'un email d'un gestionnaire pose problème, on spammerait tous les autres gestionnaires tous les jours.

Reply:
P-e mis ces lignes dans un `if !canteen.email_no_diagnostic_first_reminder` pour éviter modifier l'objet plusieurs fois ?

Reply:
Fix ici : https://github.com/betagouv/ma-cantine/pull/1363/commits/33b72196a891234cce6a81438028f731caa70df8